test/scale-infra/README.md

```diff
@@ -3,14 +3,25 @@ A scalable Kubernetes-based testbed for KubeStellar Performance Tests
 
 <img src="images/ks-scale-test-infra.drawio.svg" width="60%" height="60%" title="ks-scale-test-infra">
 
+
 ### Prerequisites
-- Ansible, with 
-  - the `amazon.aws` collection installed (see, e.g., https://docs.ansible.com/ansible/latest/collections/amazon/aws/ec2_instance_module.html), and
-  - [PyPI package boto3](https://pypi.org/project/boto/) installed for the Python interpreter that Ansible uses on your machine;
-- AWS CLI;
+
+In order to follow the instructions below, you must have [python3](https://www.python.org/downloads/) with all the dependencies listed [here](common/requirements.txt) installed. We recommend to create a python virtual environment `.venv` under `kubestellar/test/scale-infra`, for example: 
+
+```bash
+cd kubestellar/test/scale-infra 
+python3 -m venv .venv
+. .venv/bin/activate
```

Comment:
FYI, I essentially always get a complaint about pip being out of date, so the first thing I do in a venv is

```
pip install --upgrade pip
```

Reply:
ok, I see! Fixed, just added the above command in the instructions.